e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -55,6 +57,23 @@ describe("banner", () => {
       });
   });
 
+  // Until we enable multi-browser support, this repro will be skipped by Cypress in CI
+  // Issue was specific to Firefox only - it is still possible to test it locally
+  it(
```

Comment:
This is the new test that's been added. It only runs on Firefox since this issue appears on Firefox. We can verify that this test asserts the problem correctly by commenting out this line 
https://github.com/metabase/metabase/blob/38c95b047beddb1c9da00f9d845e1d00b8d41c1e/frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx#L83 and see that the test should fail.